apps/test/netsim/NetSimRouterNode.js

```diff
@@ -462,18 +462,147 @@ describe("NetSimRouterNode", function () {
         router.tick({time: 30});
         sendMessageOfSize(40);
 
-        // At t=40, the first message is sent
-        router.tick({time: 39});
-        assertTableSize(testShard, 'logTable', 0);
-        router.tick({time: 40});
+        // Jumping to t=80, we see that only one message is sent.  If we'd
+        // been using optimistic scheduling, we would have sent both.})
+        router.tick({time: 80});
         assertTableSize(testShard, 'logTable', 1);
```

Comment:
when i see a comment this long in a PR i wonder if it should just be a comment in the code :)